applications/openshift/kubelet/kubelet_enable_streaming_connections/rule.yml

```diff
@@ -32,6 +30,13 @@ rationale: |-
 
 severity: medium
 
+{{%- if product == "eks" %}}
+{{%- set kubeletconf_path = "/etc/kubernetes/kubelet/kubelet-config.json" %}}
```

Comment:
It is not clear to me where `kubeletconf_path` is used.

Reply:
thanks, just had it removed